mono/mini/interp/transform.c

```diff
@@ -3787,12 +3787,10 @@ generate (MonoMethod *method, RuntimeMethod *rtm, unsigned char *is_bb_start, Mo
 				td.sp -= 3;
 				td.ip += 1;
 				break;
-#if 0
 			case CEE_NO_:
 				/* FIXME: implement */
```

Comment:
Weirdly, the semantics of `no.{null,range,type}check` say “the subsequent instruction need not perform the specified fault check when it is executed”. So we never *need* to support this, and in fact it’s probably better not to.